db/tables.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE public.events
+(
+    id              uuid                        NOT NULL DEFAULT public.gen_random_uuid(),
+    type            character varying,
+    global_position bigserial                   NOT NULL,
+    context         character varying           NOT NULL,
+    stream_name     character varying           NOT NULL,
+    stream_id       character varying           NOT NULL,
+    stream_revision bigint                      NOT NULL,
+    data            jsonb,
+    metadata        jsonb,
```

Comment:
What about custom metadata? Didn't we decide we need it too?

Reply:
We don't need it, since metadata and custom metadata split is a EventStore DB implementation. They store there timestamp and event type for example. In our implementation we don't need that, because it is more efficient to just put that information into separated columns(`type` and `created_at`)